pkg/addons/client.go

```diff
@@ -36,27 +32,27 @@ type client struct {
 	dryRun bool
 }
 
-func newClient(dryRun bool) (*client, error) {
+func newClient(dryRun bool) (*client, *rest.Config, error) {
```

Comment:
restconfig is already a field in the returned client object, are these changes necessary?

Reply:
I didn't realize that.  removing changes.